packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.scss

```diff
@@ -92,3 +92,10 @@
 .text-footer-automatedEmail {
   @extend .text-footer;
 }
+
+.text-footer-appBadges div {
```

Comment:
Do we need this class and div at all if we shouldn't render that "install on a desktop device" text? I think we can remove it. 🤔